src/main/java/network/brightspots/rcv/RawContestConfig.java

```diff
@@ -29,7 +29,7 @@
 
 @SuppressWarnings("WeakerAccess")
 @JsonIgnoreProperties(ignoreUnknown = true)
-public class RawContestConfig {
+class RawContestConfig {
```

Comment:
Removing the `public` keywords throughout this class breaks loading config files. Please revert all changes in this file.

I recommend doing a full (manual) end-to-end test each time you're putting together a PR with big changes like this, in addition to all the automated tests. This entails booting up the GUI, loading a config file, running the tabulation, and checking that the results look as expected.

Reply:
Reverted this one, but I'd appreciate a manual test if you have the time, or else a pointer to instructions on exactly how to run it.

Reply:
It looks like you only reverted this one line, but the whole file needs to be reverted (just verified via a manual test that it's still broken).

Test manually by running the GUI via Gradle (I just click "Tasks => application => run" on the right panel in IntelliJ). Then load one of the test config files (e.g. `...\rcv\src\test\resources\network\brightspots\rcv\test_data\2017_minneapolis_mayor\2017_minneapolis_mayor_config.json`), save a copy via the Tabulator to make sure that functionality works. Re-load it. Then tabulate with it and verifying the summary JSON looks like the expected one in the same file path.

Reply:
Reverted the rest and tested manuall.y